akka-docs-dev/rst/scala/code/docs/stream/CompositionDocSpec.scala

```diff
@@ -137,17 +137,17 @@ class CompositionDocSpec extends AkkaSpec {
     //#partial-flow-dsl
     // Convert the partial graph of FlowShape to a Flow to get
     // access to the fluid DSL (for example to be able to call .filter())
-    val flow = Flow.wrap(partial)
+    val flow = Flow.fromGraph(partial)
 
     // Simple way to create a graph backed Source
-    val source = Source() { implicit builder =>
+    val source = Source.fromGraph( FlowGraph.create() { implicit builder =>
       val merge = builder.add(Merge[Int](2))
       Source.single(0)      ~> merge
       Source(List(2, 3, 4)) ~> merge
 
       // Exposing exactly one output port
-      merge.out
-    }
+      SourceShape(merge.out)
+    })
```

Comment:
The idea with the change is to make the power-level distinction between the FlowGraph-builder and "normal" Flow/Source/Sink-combinators clear and reduce the surface area of the FlowGraph-related APIs.

The pattern is consistent: <Type>.fromGraph( FlowGraph.create() { implicit builder => <Type>Shape(…) }

I'd love to see alternate solutions to reduce the surface area of the FlowGraph API and make the power-level distinction clearer.